app/src/main/res/values/themes.xml

```diff
@@ -27,6 +27,10 @@
         <item name="colorOnSurfaceInverse">@color/md_theme_light_inverseOnSurface</item>
         <item name="colorSurfaceInverse">@color/md_theme_light_inverseSurface</item>
         <item name="colorPrimaryInverse">@color/md_theme_light_primaryInverse</item>
+
+        <item name="android:navigationBarColor">
+            #B3000000
```

Comment:
What do you think about using a color variable here?

Reply:
Forgot to reply here earlier, but I had just left this here as a fallback for anything below Android 10. Was removed in later commits as it wasn't necessary.